.evergreen.yml

```diff
@@ -316,6 +316,18 @@ functions:
         content_type: text/css
         permissions: public-read
         preserve_path: true
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_files_include_filter:
+          ["spruce/storybook-static/**/*.json"]
+        remote_file: spruce/${task_id}/storybook/
+        bucket: mciuploads
+        content_type: application/json
+        permissions: public-read
+        preserve_path: true
   
```

Comment:
It looks like artifacts aren't being uploaded to the right place. We are missing images in the uploaded stories and the fonts don't seem to be picked up
<img width="918" alt="image" src="https://user-images.githubusercontent.com/4605522/227048840-ab0adb8d-a7ca-40da-a798-506eccffa276.png">